### PR TITLE
Feature/projects

### PR DESCRIPTION
**Pull request by mbollemeijer**
- Adds a projects feature

screen shot 2018-07-06 at 14 31 06
screen shot 2018-07-06 at 14 31 22

I am not a less/css magician so if someone want to improve stuff be my guest :-)

### CODE REVIEW

File: static/less/style.less [hunk 1]
```diff
@@ -1,4 +1,8 @@
-@import "colors.less";
```
Comment (luizdepra) [inline static/less/style.less]: We got regressions all over this file. Maybe a merge messed up things.

**Comment by luizdepra** [comment]
Nice!
This is one of the features I'll trying to bring to the v2 of Hugo Code.

But, we need to change the grid to work on small mobile screens. Maybe one project per line?
If you're not comfortable to do this change, I can accept this PR and do it later.

File: static/less/style.less (hunk 1 above)
Comment (mbollemeijer) [reply static/less/style.less]: I will see if I can fix this, thought I merged correctly.

**Comment by mbollemeijer** [comment]
This is what I get when I scale my browser, I must admit I have not tested it anyother way.   

screen shot 2018-07-09 at 10 02 21

If this is not what it should look like, could you provide an example so I have a reference.

File: static/less/style.less (hunk 1 above)
Comment (mbollemeijer) [reply static/less/style.less]: This should be fixed now

**Comment by mbollemeijer** [comment]
@luizdepra 
Something like this ?
screen shot 2018-07-09 at 14 47 39

**Comment by luizdepra** [comment]
The left margin of the grid items is not ok. It leaves a weird small gap between the alignment of the title and the grid.
I don't how to fix this problem yet. Hope we can find a solution together.

**Comment by mbollemeijer** [comment]
If someone could pickup my slack, that would be great.
Havent got the time sadly :-(

**Comment by luizdepra** [comment]
No problem. I will finish this feature later.

**Comment by DewaldDeJager** [comment]
@luizdepra May I pick this up? Is it just the issues raised above that need to be tended to?

**Comment by luizdepra** [comment]
Yep. But this PR was opened before a major refactor. Probably is a good idea to start from zero.

**Comment by luizdepra** [comment]
We should redo this change in another PR.
There no point in keeping this PR open to never merge it.

**Pull request closed without merging**